cmd/ctr/import.go

```diff
@@ -6,31 +6,62 @@ import (
 	"os"
 
 	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/contrib/docker1"
+	"github.com/containerd/containerd/imageformats"
+	oci "github.com/containerd/containerd/imageformats/oci"
 	"github.com/containerd/containerd/log"
 	"github.com/urfave/cli"
 )
 
 var imagesImportCommand = cli.Command{
-	Name:        "import",
-	Usage:       "import an image",
-	ArgsUsage:   "[flags] <ref> <in>",
-	Description: `Import an image from a tar stream.`,
-	Flags: []cli.Flag{
+	Name:      "import",
+	Usage:     "import an image",
+	ArgsUsage: "[flags] <in>",
+	Description: `Import an image from a tar stream.
+Implemented formats:
+- oci.v1     (default)
+- docker.v1
+`,
+	Flags: append([]cli.Flag{
 		cli.StringFlag{
-			Name:  "ref-object",
-			Value: "",
-			Usage: "reference object e.g. tag@digest (default: use the object specified in ref)",
+			Name:  "format",
+			Value: "oci.v1",
+			Usage: "image format. See DESCRIPTION.",
+		},
+		cli.StringFlag{
+			Name:  "docker.v1-prefix",
+			Value: "imported/docker.v1/",
```

Comment:
Namespaces are not to used as types. They are namespaces. What namespace will these images be imported under?

Reply:
The reason I introduced this prefix is solely because the image name spec requires some "locator" string and not intended to be interpreted as a nanespace-like stuff.


If we can relax name spec, we can omit this prefix